src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) {
+        String textFrom1 = "supply,30\n"
+                + "buy,10\n"
+                + "buy,13\n"
+                + "supply,17\n"
+                + "buy,10\n"
+                + "supply,5\n"
+                + "supply,80\n"
+                + "buy,39\n"
+                + "supply,56\n"
+                + "buy,32\n"
+                + "buy,11\n";
```

Comment:
Large methods are very difficult to read and understand. It's better to divide the logic into separate methods. For example, the logic for creating sample data could be in its own method.